### PR TITLE
expired locked rules probe ported to SQLAlchemy

### PR DESCRIPTION
**Pull request by FernandoGarzon**

### CODE REVIEW

File: common/check_expired_locked_rules [hunk 1]
```diff
@@ -6,51 +6,96 @@
...
+    """
     status = OK
     session = get_session()
     try:
-        query = "select rawtohex(id), scope, name, rse_expression from atlas_rucio.rules where locked=1 and expires_at<sys_extract_utc(localtimestamp)"
-        print 'Locked expired rules'
-        for row in session.execute(query):
+        print('Locked expired rules')
+        query = (session.query(models.ReplicationRule.id, models.ReplicationRule.scope, 
+        models.ReplicationRule.name, models.ReplicationRule.rse_expression))
```
Comment (ericvaandering) [inline common/check_expired_locked_rules:45]: Please format that correctly. PyCharm does a nice job or just make sure the various "models" line up under each other and that the filters line up under session.

File: common/check_expired_locked_rules [hunk 2]
```diff
@@ -6,51 +6,96 @@
...
     try:
-        query = """select rawtohex(c.id), c.scope, c.name, c.rse_expression from atlas_rucio.rules c,
-                   (select a.scope, a.name from atlas_rucio.dids a
-                   where a.expired_at is not null and a.expired_at < sys_extract_utc(localtimestamp)
-                   and exists (select 1 from atlas_rucio.rules b where a.scope=b.scope and a.name=b.name and locked=1)) d
-                   where c.scope=d.scope and c.name=d.name and locked=1"""
-        print 'Datasets expired with locked rules'
-        for row in session.execute(query):
+        print('Datasets expired with locked rules')
+        query = (session.query(models.ReplicationRule.id,models.ReplicationRule.scope,
```
Comment (ericvaandering) [inline common/check_expired_locked_rules:69]: Same thing. Format these nicely

File: common/check_expired_locked_rules [hunk 3]
```diff
@@ -6,51 +6,96 @@
...
+        .join(models.DataIdentifier, (models.ReplicationRule.scope == models.DataIdentifier.scope)
+                           & (models.ReplicationRule.name == models.DataIdentifier.name))
+                     .filter(models.ReplicationRule.locked == True)
+                     .filter(models.DataIdentifier.expired_at != None)
+                     .filter(models.DataIdentifier.expired_at < datetime.utcnow()))
+        for row in query.all():
             status = CRITICAL
-            print row[0], row[1], row[2], row[3]
+            id = unicode(row[0])
+            print(id, row[1], row[2], row[3])
```
Comment (ericvaandering) [inline common/check_expired_locked_rules:79]: @dchristidis do we gain anything by printing out the results of these queries? CMS is only using this to feed to monitoring, so we don't care about it.

**Review by ericvaandering — COMMENTED** [review]
@dchristidis asking for your review too since this is part of the common probes

**Review by ericvaandering — CHANGES_REQUESTED** [review]
Please see comments inline

File: common/check_expired_locked_rules (hunk 3 above)
Comment (ofgarzon2662) [reply common/check_expired_locked_rules:79]: Printing is on the old probes, so I just copied that. Should I delete those? @dchristidis

File: common/check_expired_locked_rules (hunk 1 above)
Comment (FernandoGarzon) [reply common/check_expired_locked_rules]: Does it look better now?

**Pull request closed without merging**